### DISCUSSION TITLE
Como praticar para FrontEnd, inicialmente?

### DISCUSSION THREAD

**Discussion by jhongalante** [frontendbr/forum#1279] [Discussão]
Boa tarde a todos.

Estou iniciando meus estudos no FrontEnd, ja até fiz um curso na Udemy, porém quero começar a fazer os projetos la do inicio, apenas com HTML/CSS. Porém não sei bem como fazer para praticar, sabe? 
Tinha pensado em pegar alguns templates e tentar montar com html/css, porém assim eu fiquei meio perdido em como pegar as imagens que são utilizadas, por exemplo, dentre outras coisas...

Como vocês fizeram/aconselham pra quem está iniciando?

**Comment by nazarepiedady** [comment]
Procure tentar criar elementos de interfaces, como botões, links, cards, barra de busca, e depois vai crescendo criando sites completos, mas procure faze-los cada vez mais bonitos a medida que vai crescendo.

**Comment by ViniciusGularte** [comment]
Seguindo a dica do @nazarepiedady , tem uma plataforma muito para isso a  [Codepen](https://codepen.io/#),onde você desenvolve e já visualiza o que está fazendo, além de poder ver o que o pessoal ta criando(Sempre tiro ideia de lá quando to sem criatividade).

Outra dica é essa plataforma [FreeCodeCamp](https://www.freecodecamp.org/) você pode realizar os desafios deles, isso só não aumenta teu portfólio como no final de cada modulo você ganha um certificado por isso.

Pega algum teste de empresa e tenta replicar, buscando como fazer as coisas, a gente sempre aprende algo,  para isso recomendo esse repositório [FrontEnd Challenges](https://github.com/LFeh/frontend-challenges) .

Minha ultima dica é se atentar as boas praticas, endentação e esse tipo de coisa, criar o habito de tornar teu código legível e limpo desde o começo já facilita muito as coisas pro futuro.

:wink:

**Comment by andersoonweb** [comment]
@JhonGalante, eu vou até escrever um texto e publicar no medium sobre esse assunto.
Na thead #1148 eu dou algumas dicas de que realmente eu acredito.
Como um _plus_ posso acrescentar um dica que foi me passada há alguns dias: A leitura de documentações é **EXTREMAMENTE IMPORTANTE**. 
Você pode até ver cursos, não há nada de errado nisso - Eu adoro!. Porém, beba sempre direto da fonte, nem que seja dá um olhadinha; uma leitura rápida.
As vezes pode ser chato, mas é necessário. **Sempre beba água direto da fonte**, principalmente no início. Chega uma hora que você não vai ter como achar curso, textos, enfim, vai se acostumando.

PS: Eu já tinha a perspectiva dessa dica, porém, nunca refletir. Obrigado @neysimoes.

Espero te ajudar. Bons estudos!

**Comment by DevAlanAraki** [comment]
Muito boa as dicas pessoal. Estou retornando para a área de Front End, estou à fazer um curso para me atualizar, pois sai da área há cerca de 8 anos e muitas coisas evoluíram desde lá. Assim, estou buscando essa área aqui na Irlanda que sempre foi minha paixão e aqui se tem bons rendimentos financeiros nessa área. A pergunta do amigo Jhon é válida para muitos que estão, ou começando, ou voltando para esta área.

Abraços.

**Comment by julombe** [comment]
Viva pessoal.
Também estou entrando agora na area de desenvolvimento front-end assim como @JhonGalante.
Até ao momento para praticar encontrei o codepen e achei muito bom. Os cursos da @Alura também ajudam muito. Boa caminhada para vc...
